src/main/js/utils/ScheduleMaker.js

```diff
@@ -2,15 +2,16 @@
 import Schedule from './Schedule';
 
 class ScheduleMaker {
-  constructor(courseListings, gap) {
+  constructor(courseListings, gap, sort) {
```

Comment:
By removing `sort` from `Schedule` it can also be removed from here

Reply:
Removed.